Formula/ponyc.rb

```diff
@@ -1,10 +1,9 @@
 class Ponyc < Formula
   desc "Object-oriented, actor-model, capabilities-secure programming language"
   homepage "http://www.ponylang.org"
-  url "https://github.com/ponylang/ponyc/archive/0.2.1.tar.gz"
-  sha256 "cb8d6830565ab6b47ecef07dc1243029cef962df7ff926140022abb69d1e554e"
-  revision 1
-  head "https://github.com/ponylang/ponyc.git"
+  url "https://github.com/ponylang/ponyc.git",
+    :revision => "aafebac938273d4786c02dfb2ba9ec7a164675e7"
+  version "0.2.2-candidate"
```

Comment:
If you experiment in `brew irb`: we don't use simple string comparison for versions. You'll have to compare `Version.new("0.2.2")` with `Version.new("0.2.2-candidate")`. How formula versions compare with each other is implemented by Homebrew and we have code in place that sorts `alpha`, `beta`, `rc` (and often used variations of these) before the version number without suffix (the stable release), so that versions are sorted like most people would expect them to be.